### PR TITLE
Add Okta CLI to QuickStart Guides

### PR DESCRIPTION
**Pull request by mraible**
Reverts https://github.com/okta/okta-developer-docs/pull/1521 and cleans things up a bit.

## Description:
- **What's changed?** Add the Okta CLI
- **Is this PR related to a Monolith release?** No.

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/quickstart/create-org/website/create-org.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+2. Fill out the form. You'll need to supply:
+
+    - First name
+    - Last name
+    - Email address
+    - Company
+
+3. Click **Get Started**.
+
+4. You receive an email to activate your account.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/quickstart/create-org/website/create-org.md:14]:
```suggestion
4. Click the link in the email that you receive to activate your account.
```

File: packages/@okta/vuepress-site/docs/guides/quickstart/register-app/cli/register-app.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+To use the Okta CLI:
+
+1. Enter the command: `okta apps create`
+2. Give your app a name.
+3. Select **SPA** (for Angular, React or Vue) or **Web** for Type of Application.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/quickstart/register-app/cli/register-app.md:5]:
```suggestion
3. Select **SPA** (for Angular, React or Vue) or **Web** as the  **Type of Application**.
```

File: packages/@okta/vuepress-site/docs/guides/quickstart/register-app/cli/register-app.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+4. Select **Other**.
+5. For Login redirect URI, enter: `http://localhost:8080/login/callback`
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/quickstart/register-app/cli/register-app.md:7]:
```suggestion
5. For the **Login redirect URI**, enter: `http://localhost:8080/login/callback`.
```

File: packages/@okta/vuepress-site/docs/guides/quickstart/register-app/cli/register-app.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+To use the Okta CLI:
+
+1. Enter the command: `okta apps create`
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/quickstart/register-app/cli/register-app.md:3]:
```suggestion
1. Enter the command: `okta apps create`.
```

File: packages/@okta/vuepress-site/docs/guides/quickstart/create-org/website/create-org.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+You can create an account on our website:
+
+1. Go to: <https://developer.okta.com/signup>
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/quickstart/create-org/website/create-org.md:3]:
```suggestion
1. Go to: <https://developer.okta.com/signup>.
```

File: packages/@okta/vuepress-site/docs/guides/quickstart/register-app/index.md [hunk 6]
```diff
@@ -27,31 +27,11 @@ To start, find the example applications provided by Okta on GitHub for the langu
 
 Specifically, look for the `okta-hosted-login` example (there are samples provided for several use cases).
 
-You can try building the example app as-is to start or you can use it as a template for modifying your own app. When you have an app running, you're ready to connect it to your Okta org. 
+You can try building the example app as-is to start or you can use it as a template for modifying your own app. When you have an app running, you're ready to connect it to your Okta org.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/quickstart/register-app/index.md:30]:
```suggestion
You can try building the example app as-is to start, or you can use it as a template for modifying your own app. When you have an app running, you're ready to connect it to your Okta org.
```

File: packages/@okta/vuepress-site/docs/guides/quickstart/register-app/website/register-app.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+1. Click **Applications**.
+
+2. Click **Add Application**.
+
+3. Select **Web** as the type of application. Click **Next**.
+
+4. Enter the **Name** of your app.
+
+5. Leave **Base URI** set to the default. This is the domain where your app runs. For this quick start guide, we'll use `localhost`, which is where the sample apps run by default. You might need to change the port, depending on the default port your framework uses to serve `localhost` pages.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/quickstart/register-app/website/register-app.md:11]:
```suggestion
5. Leave **Base URI** set to the default. This is the domain where your app runs. For this quick start guide, we use `localhost`, which is where the sample apps run by default. You might need to change the port, depending on the default port your framework uses to serve `localhost` pages.
```

File: packages/@okta/vuepress-site/docs/guides/quickstart/register-app/website/register-app.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+
+6. For **Login redirect URIs**, enter: `http://localhost:8080/login/callback`. Okta sends OAuth authorization responses to the specified URIs, which are also known as callback endpoints.
+
+5. Leave **Logout redirect URIs** set to the default. This setting lets you specify a URI to redirect the user’s browser to when they sign out.
+
+6. Leave **Group assignments** set to the default, which is "Everyone". Users can only sign in to apps that they are assigned to. Group assignments let you manage assignments for large numbers of users at once.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/quickstart/register-app/website/register-app.md:17]:
```suggestion
6. Leave **Group assignments** set to the default, which is **Everyone**. Users can only sign in to apps that they are assigned to. Group assignments let you manage assignments for large numbers of users at once.
```

File: packages/@okta/vuepress-site/docs/guides/quickstart/register-app/website/register-app.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+
+7. For **Grant type allowed**, under "Client acting on behalf of a user", select **Authorization Code**. This is the OAuth 2.0 grant type that Okta will provide.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/quickstart/register-app/website/register-app.md:19]:
```suggestion
7. For **Grant type allowed**, under "Client acting on behalf of a user", select **Authorization Code**. This is the OAuth 2.0 grant type that Okta provides.
```

**Review by susanharper-okta — COMMENTED** [review]
couple things. thanks!

File: packages/@okta/vuepress-site/docs/guides/quickstart/create-org/cli/create-org.md [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+2. You are prompted to supply the following information:
+
+    - First name
+    - Last name
+    - Email address
+    - Company
+
+3. Okta creates the org. The CLI returns your Okta domain and lets you know that a verification code is being sent to your email.
+
+   Make a note of your Okta domain: It’s the base of the URL you use to access your org. Authorization requests for users will be directed to an endpoint that has this as its base, and any Okta API endpoints you call will also have this URL as their base.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/quickstart/create-org/cli/create-org.md:14]: Do we want this to be a > **Note:** as well?

Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/quickstart/create-org/cli/create-org.md:14]:
```suggestion
> **Note:** Make a note of your Okta domain: It’s the base of the URL you use to access your org. Authorization requests for users will be directed to an endpoint that has this as its base, and any Okta API endpoints you call will also have this URL as their base.
```

**Review by jakubvul-okta — APPROVED** [review]

**Comment by mraible** [comment]
@jakubvul-okta @susanharper-okta Please merge at your convenience. I don't have permission.

**Pull request merged**